src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -89,12 +40,13 @@ static ZipInputStream noOpVerification(ZipStreamWithSignature zipWithSignature)
         return zipWithSignature.zipInputStream;
     }
 
-    static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSignature)
-        throws DocSignatureFailureException {
+    /**
+     * Checks whether the wrapping zip has expected entries and returns the internal zip.
+     */
+    static ZipInputStream verifyAndExtract(ZipStreamWithSignature zipWithSignature) {
         Map<String, byte[]> zipEntries = extractZipEntries(zipWithSignature.zipInputStream);
 
         verifyFileNames(zipEntries.keySet());
```

Comment:
we are verifying the filenames in blob router. are you planning to remove this in the next PR?

Reply:
good point, I've removed that check as well.